legacy-content-import/src/main/scala/legacycontentimport/Article.scala

```diff
@@ -27,17 +37,30 @@ object Article {
 
   private def fromCapiHelpSectionPage(capiDomain: String, capiKey: String)(pageIndex: Int): Seq[Article] = {
 
-    def toArticle(result: ujson.Value): Article = Article(
-      title = result("webTitle").str,
-      url = new URI(result("webUrl").str),
-      publicationDate = Instant.parse(result("webPublicationDate").str)
-    )
+    def toArticle(result: ujson.Value): Article = {
+
+      val tagsToInclude = result("tags").arr.toList.filterNot(tag => excludedTags.contains(tag("id").str))
+
+      def tagsOfType(typeName: String) =
+        tagsToInclude.filter(tag => tag("type").str == typeName).map(_("webTitle").str)
```

Comment:
Maybe combine filter and map into collect, something like

```
tagsToInclude collect { case tag if tag("type").str == typeName => tag("webTitle").str }
```

Reply:
Done.